modules/client-configuration/pages/autoinst-profiles.adoc

```diff
@@ -47,45 +66,176 @@ For more information about autoinstallation distributions and files, see xref:cl
+Alternatively, navigate to menu:Systems[System List], select the client you want to provision, and navigate to the menu:Provisioning[Autoinstallation] subtab.
+
+
+[[autoyast]]
+== AutoYast Profiles
+
+An {ay} profile consists of a [guimenu]``Label`` that identifies the profile, an [guimenu]``Autoinstall Tree`` that points to an autoinstallation distribution, various options, and, most importantly, an {ay} installation file.
 
-To adjust advanced autoinstallation configuration options, navigate to menu:Systems[System List], select the client you want to provision, and navigate to the menu:Provisioning[Autoinstallation] subtab.
-Select the autoinstallation profile to use, and click btn:[Advanced Configuration].
-For traditional clients, use the various fields to set options for specialized {kickstart} scenarios.
-For Salt clients, provide the kernel options you require.
-For more information about these options, see https://documentation.suse.com/sles/html/SLES-all/book-sle-admin.html.
+The {ay} installation file is an XML file that give directions to the {ay} installer. {ay} calls it a "control file".
+For the full syntax of {ay} installation files, see https://doc.opensuse.org/projects/autoyast/#cha-configuration-installation-options.
 
+{suse} provides templates of {ay} installation files that you can use as a starting point for your own custom files.
+You will find them at https://github.com/SUSE/manager-build-profiles.
+Each of these profiles requires you to set some variables before you use it.
+Check the [path]``README`` file included with the script to determine which variables you need.
+For more information about using variables in {ay} scripts, see xref:client-configuration:autoinst-profiles#variables[Variables].
 
+For more information about {ay}, see https://doc.opensuse.org/projects/autoyast/.
 
-== File Preservation for {kickstart}
+A more recent, Salt-based alternative to AutoYast, is Yomi.
+For information about Yomi, see xref:salt:yomi.adoc[Install with Yomi].
 
-If you have many custom configuration files located on a client you want to {kickstart}, you can save them as a list, and associate that list with the {kickstart} profile to be used.
 
-In the {productname} {webui}, navigate to menu:Systems[Autoinstallation > File Preservation] and click btn:[Create File Preservation List].
-Enter a suitable label, and list absolute paths to all files and directories you want to save.
-Click btn:[Create List].
+[[kickstart]]
+== Kickstart Profiles
 
-Include the file preservation list in your {kickstart} profile by navigating to menu:Systems[Autoinstallation > Profiles] and selecting the profile you want to edit.
-Navigate to the menu:System Details[File Preservation] subtab and select the file preservation list to include.
+{kickstart} profiles offer a larger number of configuration options.
```

Comment:
```suggestion
{kickstart} profiles offer a large number of configuration options.
```

"Larger" sounds like "Kickstart is better than Autoyast as it provide more config options". AFAIK the oposit is true

Reply:
It's simply a typo - my bad.